### ISSUE TITLE
why not support INESRT INTO .....   ON DUPLICATE KEY UPDATE  use the dynamic parameter

### ISSUE THREAD

**Issue by RogerLuoZJ** [sharding-sphere/sharding-sphere#1388]
As i use the sharding-sphere in my old project . There is some sql segement like 
insert into a (a,b) value(?,?) on duplicate key update a= ? , b = ? + b by use the PreparedStatement
then the sharding-sphere will ignore the last two question symbol , then will cause the parameter 3 not found exception . 
As i go through the source code in version 3.0.1-SNAPSHOT and found there is not any logic to handle the placeholder in 'on duplicate key update' , so just support sql like insert into a (a,b) value(?,?) on duplicate key update a= VALUES(a) , b = VALUES(b) .
Does this feature will support sooner , or any consideration for the sql at the beginning.
As i have modified the code to support it .

**Comment by ma-xiao-guang-64** [comment]
Thank you, you are right.
[like issue](https://github.com/sharding-sphere/sharding-sphere/issues/1066)

Why sharding-sphere can't support this?
1、It will increase the complexity of the code
2、You can use VALUES(column) instead of it
3、We are developing antlr sql parser. when this is done, we will redesign parser engine

**Comment by RogerLuoZJ** [comment]
how i use one sql to replace the sql  'insert into a (a,b) value(?,?) on duplicate key update a= ? , b = ? + b '  by VALUES(column) ?

**Comment by ma-xiao-guang-64** [comment]
@RogerLuoZJ  yes, [like issue](https://github.com/sharding-sphere/sharding-sphere/issues/1066)
